src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboardOverview.tsx

```diff
@@ -127,9 +132,14 @@ export function GovernanceDAODashboardOverview() {
           </GridItem>
         </Grid>
       </Flex>
-      <Flex gap="4" direction="column">
-        <Text textStyle="h4 medium">Recent Transactions</Text>
-      </Flex>
+      <RecentProposals
+        proposals={proposals?.sort((proposalA, proposalB) => +proposalB.timestamp - +proposalA.timestamp).slice(0, 3)}
```

Comment:
I recommend moving display calculation/business logic above the return function within React components: 

`const recentProposals = proposals?.sort((proposalA, proposalB) => +proposalB.timestamp - +proposalA.timestamp).slice(0, 3)`

This helps with component maintenance and debugging as a component grows in complexity.

Reply:
Thanks for the feedback. made the suggested changes.